option.go

```diff
@@ -282,6 +302,60 @@ func (option *Option) Set(value *string) error {
 	return convert("", option.value, option.tag)
 }
 
+// SetTerminated sets the value of a "terminated" option to the specified value.
+// An error will be returned if the specified value could not be
+// converted to the corresponding option value type.
+// If the option value is a slice of slices, each element in the parameter list
+// (excluding the terminator) is converted and the resulting list is appended to the option value.
+// Otherwise, each element is converted and the option value is set to
+// the resulting list.
+func (option *Option) SetTerminated(value []string) error {
+	tp := option.value.Type()
+
+	if tp.Kind() != reflect.Slice {
+		return newErrorf(ErrInvalidTag,
+			"terminated flag `%s' must be a slice or slice of slices",
+			option.shortAndLongName())
+	}
+
+	if option.clearReferenceBeforeSet {
+		option.empty()
+	}
+
+	option.isSet = true
+	option.preventDefault = true
+	option.clearReferenceBeforeSet = false
+
+	elemTp := tp.Elem()
+
+	if elemTp.Kind() == reflect.Slice {
+		var elemVal reflect.Value
+
+		if len(value) > 0 {
+			elemVal = reflect.Indirect(reflect.New(elemTp))
+			for _, val := range value {
+				if err := convert(val, elemVal, option.tag); err != nil {
```

Comment:
That's quite a confusing API. It's not just converting, but appending the item. Anyway, not something we need to do something about right now.